src/Component/Renderer/OlRenderer/placeholder.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable max-len */
+/**
+ * This is the base64 datauri representation of the placeholder image.
+ * The "FunctionOutlined" icon from antd was the source. Compare ./placeholder.svg
```

Comment:
```suggestion
 * The "FunctionOutlined" icon from antd (MIT-licensed) was the source. Compare ./placeholder.svg
```

See https://github.com/ant-design/ant-design-icons I think it makes sense to mention the license.